### PR TITLE
Addition of variables related to power plants from openTEPES

### PR DESCRIPTION
**Pull request by erikfilias**
Two commits:

- Moving variables related to CO2 emissions of power plants from power-plant.yaml to emissions.yaml
- Reorder and addition of variables in power-plant.yaml

### CODE REVIEW

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 1]
```diff
@@ -464,12 +630,45 @@ Inertia|Electricity|Hydro|Reservoir:
...
+   description: This is the upper bound that energy storage system could charge energy at time "t".
+   unit: GWh
+
+Initial Storage Level|Electricity|Energy Storage System:
+   description: Initial energy stored in the energy storage system unit at first load level. (ESS that includes hydro, battery, etc.)
+   unit: GWh
+
+Maximum Charge|Electricity|Energy Storage System:
+   description: Maximum charge when storing energy the energy storage system unit.
+   unit: MW
```
Comment (sebastianzwickl) [inline nomenclature/definitions/variable/technology/power-plant.yaml:651]: Related to the conversation at https://github.com/openENTRANCE/nomenclature/issues/50#issuecomment-642560593 I suppose to add **Maximum Discharge**.

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 2]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
                 (IPCC category 3)
    unit: Mt CO2/yr
 
+Emmissions|CO2|Electricity:
```
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml:293]: Emmissions|CO2|Electricity:
```suggestion
Emissions|CO2|Electricity:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 3]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
...
+   description: CO2 emmissions of power plants .
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Biomass|Traditionnal:
```
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml:297]:
```suggestion
Emissions|CO2|Electricity|Biomass|Traditionnal:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 4]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
...
+   description: CO2 emmissions of a typical Traditionnal Biomass power plant
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Biomass|New and imported :
```
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml:303]:
```suggestion
Emissions|CO2|Electricity|Biomass|New and imported :
```

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 5]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
...
+   description: CO2 emmissions of a typical new biomass power plant
+                biomass from dedicated energy crops and biomass imports.
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Coal:
```
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml:308]:
```suggestion
Emissions|CO2|Electricity|Coal:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 6]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
...
+   description: CO2 emmissions of a typical coal power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Gas|OCGT:
```
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml:312]:
```suggestion
Emissions|CO2|Electricity|Gas|OCGT:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 7]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
...
+   description: CO2 emmissions of a typical Open Cycle Gas Turbine power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Gas|CCGT:
```
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml:316]:
```suggestion
Emissions|CO2|Electricity|Gas|CCGT:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 8]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
...
+   description: CO2 emmissions of a typical Combined Cycle Gas Turbine power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Lignite:
```
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml:320]:
```suggestion
Emissions|CO2|Electricity|Lignite:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 9]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
...
+   description: CO2 emmissions of a typical Lignite power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Nuclear:
```
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml:324]:
```suggestion
Emissions|CO2|Electricity|Nuclear:
```

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 1 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/technology/power-plant.yaml:651]: Ok, got it.

Comment (sebastianzwickl) [reply nomenclature/definitions/variable/technology/power-plant.yaml:651]: Perfect, thank you @erikfilias

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 10]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
                 (IPCC category 3)
    unit: Mt CO2/yr
 
+Emissions|CO2|Electricity:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/emissions/emissions.yaml:293]: I suggest to change this for clarification that this is not related to the consumption of electricity (and also implement for the other variables)
```suggestion
Emissions|CO2|Electricity Generation:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 11]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
...
+   description: CO2 emmissions of power plants .
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/emissions/emissions.yaml:294]: typo, please correct throughout this document and remove `.`
```suggestion
   description: CO2 emissions of power plants
```

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 12]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
...
+   unit: tons/MWh
+
+Emissions|CO2|Electricity|Biomass|Traditionnal:
+   description: CO2 emmissions of a typical Traditionnal Biomass power plant
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit: tons/MWh
+
+Emissions|CO2|Electricity|Biomass|New and imported :
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/emissions/emissions.yaml:303]: typo
```suggestion
Emissions|CO2|Electricity|Biomass|New and imported:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 13]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
                 (IPCC category 3)
    unit: Mt CO2/yr
 
+Emissions|CO2|Electricity:
+   description: CO2 emmissions of power plants .
+   unit: tons/MWh
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/emissions/emissions.yaml:295]: I am skeptical to have this variable as "per unit of generated electricity", because other `Emissions|CO2|*` variables are reporting **total emissions** - so this could easily lead to confusion.

If you want to add a variable to describe operational characteristics, can we find an alternative name?

@sandrinecharousset what do you think?

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 14]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
...
+
+Emissions|CO2|Electricity|Gas|CCGT:
+   description: CO2 emmissions of a typical Combined Cycle Gas Turbine power plant
+   unit: tons/MWh
+
+Emissions|CO2|Electricity|Lignite:
+   description: CO2 emmissions of a typical Lignite power plant
+   unit: tons/MWh
+
+Emissions|CO2|Electricity|Nuclear:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/emissions/emissions.yaml:324]: what are the CO2 emissions of a nuclear power plant during normal operations?

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 15]
```diff
@@ -1,4 +1,135 @@
-# description of thermal power plants
+# List of variables related to power plants
+# Variables related to the production cost of a power plant
+# Note that:
+# linear   term cost   [€/GWh] = Linear Var Cost [Mcal/GWh] * Price|Primary Energy [€/Mcal] + Operation and Maintenance Variable Cost [€/MWh] * 1e-3
+# constant term cost   [€/h]   = Constant Var Cost [Mcal/h] * Price|Primary Energy [€/Mcal]
+Constant Var Cost|Electricity|Coal:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/technology/power-plant.yaml:6]: suchest to rename that to
```suggestion
Operational Cost|Constant Term|Electricity Generation|Coal:
```

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 16]
```diff
@@ -1,4 +1,135 @@
...
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the unit commitment variable of each power plant per unit time.
+   unit: Mcal/hour
+
+Constant Var Cost|Electricity|Oil:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the unit commitment variable of each power plant per unit time.
+   unit: Mcal/hour
+
+Linear Var Cost|Electricity|Coal:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/technology/power-plant.yaml:26]:
```suggestion
Operational Cost|Linear Term|Electricity Generation|Coal:
```

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 17]
```diff
@@ -1,4 +1,135 @@
...
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Start-Up Cost|Electricity|Oil:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Variable Cost|Electricity|Biomass|Traditionnal:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/technology/power-plant.yaml:105]: Can these variables be removed as there is now a clear separation into linear and fixed components of the operational costs?

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 18]
```diff
@@ -44,7 +176,11 @@ Maximum Active power|Electricity|Gas|OCGT:
    description: Maximum active power of one typical Open Cycle Gas Turbine power plant
    unit: MW
 
-Maximum Active power|Electricity|Gas|Coal:
+Maximum Active power|Electricity|Gas|OCGT:
+   description: Maximum active power of one typical Combined Cycle Gas Turbine power plant
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/technology/power-plant.yaml:180]: This is not the active power of the plant but normalized to the capacity (per MW)

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 13 above)
Comment (arght) [reply nomenclature/definitions/variable/emissions/emissions.yaml:295]: For a power plant we can use emission rate

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 14 above)
Comment (arght) [reply nomenclature/definitions/variable/emissions/emissions.yaml:324]: I understand that it is just written for consistency of dealing with all the conventional power plants

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 13 above)
Comment (sandrinecharousset) [reply nomenclature/definitions/variable/emissions/emissions.yaml:295]: @danielhuppmann @arght I agree with the proposal of using emission rate instead of emission; I will include it in 'review mode' so that erik can accept changes easily (those are variables I did create in an older PR)

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 14 above)
Comment (sandrinecharousset) [reply nomenclature/definitions/variable/emissions/emissions.yaml:324]: yes this is a consistency variable.... emissions of nuclear ar 0.....In my model, as in most UC models and it may be the same in comillas, nuclear is modelled as a thermal plant then it has the same list of variables (see issue #59 that I created about the lists of technologies : there are groups of technologies that share the same list of variables); Not sure what is the best to do there

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 17 above)
Comment (sandrinecharousset) [reply nomenclature/definitions/variable/technology/power-plant.yaml:105]: yes; I think I did create those variables long ago, before erik added the more detailed ones from tepes.

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 14 above)
Comment (arght) [reply nomenclature/definitions/variable/emissions/emissions.yaml:324]: should we propose the use of t for ton following the International System of Units?

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 18 above)
Comment (sandrinecharousset) [reply nomenclature/definitions/variable/technology/power-plant.yaml:180]: this is the complex stuff here. When we model a generation mix that already exist, we can describe power plants one by one and each one has its own technical data (like its maximum active power....) and a name. When we are dealing with the future, what we do is that we describe a serie of  'typical' power plant (because we need to know what is te maximal active power of ONE single plant, as the model will include all the integer constraints....), usually one per technology per country (as countries may manage their plants differently, example: french nuclear plants are flexible and used intensively for providing ancillary services, while in most countries nuclear plants are operated only at nominal power, this can be seen in eg the minimum active power and the duration constraints) and we also describe the total capacity per technology per country. If we did not have the need of describing single units, then the Maximum Active Power variable would not be needed as it would be equal to the capacity.
To answer you comment, no this is not the normalized active power to the capacity (as this would be 1), it really something like 1400MW , 500MW...
Your comments means to me that the description should be improved.
@erikfilias any idea?

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 3 above)
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml]:
```suggestion
Emission Rate|CO2|Electricity|Biomass|Traditionnal:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 4 above)
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml]:
```suggestion
Emission Rate|CO2|Electricity|Biomass|New and imported :
```

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 5 above)
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml]:
```suggestion
Emission Rate|CO2|Electricity|Coal:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 6 above)
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml]:
```suggestion
Emission Rate|CO2|Electricity|Gas|OCGT:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 7 above)
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml]:
```suggestion
Emission Rate|CO2|Electricity|Gas|CCGT:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 8 above)
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml]:
```suggestion
Emission Rate|CO2|Electricity|Lignite:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 9 above)
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml]:
```suggestion
Emission Rate|CO2|Electricity|Nuclear:
```

File: nomenclature/definitions/variable/emissions/emissions.yaml [hunk 19]
```diff
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
...
+Emmissions|CO2|Electricity|Gas|CCGT:
+   description: CO2 emmissions of a typical Combined Cycle Gas Turbine power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Lignite:
+   description: CO2 emmissions of a typical Lignite power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Nuclear:
+   description: CO2 emmissions of a typical nuclear power plant
```
Comment (sandrinecharousset) [inline nomenclature/definitions/variable/emissions/emissions.yaml:325]:
```suggestion
   description: CO2 emmissions of a typical nuclear power plant; usually equal to 0 but variable necessary for consistancy reason
```

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 17 above)
Comment (sandrinecharousset) [reply nomenclature/definitions/variable/technology/power-plant.yaml:105]: @erikfilias I suggest we delete all variable cost| variables

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 19 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/emissions/emissions.yaml]: I've just added this explanation to the PR. 
It's committed.

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 18 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/technology/power-plant.yaml:180]: I totally agree with you, @sandrinecharousset . 
In my point of view, we could make that maximum active power and capacity will be the same because in theory data related to power generation are the same in power and energy models. 

On the other hand, I wrote maximum active power because it is a standard to define the active and reactive power in power system models. Maybe, there will be other models related to AC load flow that use the data or to be included in our project in the future. This is the reason.

In addition, I'm curious about if we will compare the total capacity of power plants used in energy models with the values that we have in power models since the first one usually use a total amount, and the second one use values per power plant, node, region, or country. 

Then, can we conclude about mixing both terms and use only the term capacity?

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 17 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/technology/power-plant.yaml:105]: Ok, I got it.

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 13 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/emissions/emissions.yaml:295]: Ok, Thanks @sandrinecharousset

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 3 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/emissions/emissions.yaml]: Thanks, I got it.

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 4 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/emissions/emissions.yaml]: Thanks, I got it.

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 5 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/emissions/emissions.yaml]: Thanks, I got it.

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 7 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/emissions/emissions.yaml]: Thanks, I got it.

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 8 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/emissions/emissions.yaml]: Thanks, I got it.

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 9 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/emissions/emissions.yaml]: Thanks, I got it.

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 14 above)
Comment (sandrinecharousset) [reply nomenclature/definitions/variable/emissions/emissions.yaml]: @arght yes I agree with t

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 18 above)
Comment (sandrinecharousset) [reply nomenclature/definitions/variable/technology/power-plant.yaml:180]: @erikfilias for my model I need both the maximum active power of one plant and the capacity of the technology. Eg I could have for nuclear in france: capacity=50000MW and maximum active power = 1400MW. Thus the model will create the good number of individual units .
then the comparison you suggest in my case is perfectly equivalent :-)

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 17 above)
Comment (sandrinecharousset) [reply nomenclature/definitions/variable/technology/power-plant.yaml:105]: @erikfilias @danielhuppmann I was a bit quick in answering. ACtually we need to keep the variable cost variable as it is not the same thing than the linear var cost. linear var cost is if I understood well a component of a formulae used by comillas to compute what I call the variable cost. In my model the cost for thermal unit is quadratic, then we have 3 inputs: quadratic cost, variable cost and fixed cost. the operation cost is then quadratic cost * active power * active power + variable cost * active power + fixed cost. As we do not have any public data available for quadratic cost and fixed cost I had only defined the variable cost. 
Maybe this is equivalent to Operation and Maintenance cost (as the unit is €/MWh) but then I would propose to define it somow differently, eg
Operation and Maintenance Cost|Linear term|Electricity|Gas
or
Operation and Maintenance Cost|Electricity|Gas|Linear term
@erikfilias I thus think that the variables like Operational Cost|Linear Term|Electricity Generation|Coal: should be named a bit differently as they are not really costs (the unit does not include €), and it is somehow ambiguous...

Comment (erikfilias) [reply nomenclature/definitions/variable/technology/power-plant.yaml:105]: Yeah, I've just checked the differences. 
In openTEPES, we make a previous step in related to your variable cost.
E.g. we consider the following:
```
Operational Cost|Linear Term|Electricity Generation|...
Operation and Maintenance Cost|Electricity|...
Operational Cost|Constant Term|Electricity Generation|...
```
And, the next expressions are calculated:
```
**linear   term cost**   [€/GWh] = Operational Cost|Linear Term|Electricity Generation|... [Mcal/MWh] * Price|Primary Energy [€/Mcal] + Operation and Maintenance Cost|Electricity|... [€/MWh] * 1e-3
**constant term cost**   [€/h]   = Operational Cost|Constant Term|Electricity Generation|... [Mcal/h] * Price|Primary Energy [€/Mcal]
```

The **linear   term cost** is equivalent to your **variable cost**, and **constant term cost** to your fixed cost, I think. 

Then, I think we could keep these  variables because are not equal. And, I agree we should to rename it.
The  `Operational Cost|Linear Term|Electricity Generation|...` is defined as the linear term (slope) of the heat rate straight line, and `Operational Cost|Constant Term|Electricity Generation|...`, as the constant term (intercept) of the heat rate straight line.
Maybe, I could propose new names based on their definitions, @sandrinecharousset, do you agree?

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 18 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/technology/power-plant.yaml:180]: Really nice! :)
Then, I will keep the variables unless there is another contrary comment.

File: nomenclature/definitions/variable/technology/power-plant.yaml (hunk 17 above)
Comment (sandrinecharousset) [reply nomenclature/definitions/variable/technology/power-plant.yaml:105]: @erikfilias yes would be more clear and avoid confusions :-) 
We also use the same kind of formulaes in our models but depending on the datasource, usually what you can find is the variable cost only..... Then in long-term models we use that one. (and in short term operational models we use the ones computed out of the formulas, but in that case we do have all the technical data)

Comment (arght) [reply nomenclature/definitions/variable/technology/power-plant.yaml:105]: I suggest to express startup and shutdown costs in MEUR instead of Mcal.

Comment (danielhuppmann) [reply nomenclature/definitions/variable/technology/power-plant.yaml:105]: what is "MEUR"? if million, "million EUR", "mEUR" might be more intuitive...

Comment (arght) [reply nomenclature/definitions/variable/technology/power-plant.yaml:105]: I suggest to keep M to be compatible with the International System of Units

File: nomenclature/definitions/variable/emissions/emissions.yaml (hunk 6 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/emissions/emissions.yaml]: I made it locally some days ago because the commit suggestion was outdated

**Comment by erikfilias** [comment]
I rebase this PR, and made one commit related to the use of "EUR".

**Comment by danielhuppmann** [comment]
merging as is and then implement a clean-up in a separate PR later
